test/machi_file_proxy_eqc.erl

```diff
@@ -130,6 +136,80 @@ check_writes(write, [{Pos, Sz}|_T], Off, L) when ( Off + L ) > Pos
 check_writes(Op, [_H|T], Off, L) ->
     check_writes(Op, T, Off, L).
 
+get_overlaps(_Offset, _Len, [], Acc) -> lists:reverse(Acc);
+get_overlaps(Offset, Len, [{Pos, Sz} = Ck|T], Acc0)
+%% Overlap judgement differnt from the one in machi_csum_table
+%% [a=Offset, b), [x=Pos, y) ...
+  when
+      %% (a-y) * (b-x)
+      %%(Offset - Pos - Sz) * (Offset + Len - Pos) < 0 ->
+      %% a x b y
+      (Offset =< Pos andalso Pos < Offset + Len andalso Offset + Len =< Pos + Sz) orelse
+      %% a x y b
+      (Offset =< Pos andalso Pos + Sz < Offset + Len) orelse
```

Comment:
The above `a x b y` case, both `a=x` and `b=y` cases are included but `x=a` is included and `y=b` is excluded in this line.
From correctness point of view, no problem. I'm thinking consistency between conditions may help reading the code afterwards.